src/ui/public/copee/copee.js

```diff
@@ -0,0 +1,46 @@
+import Notifier from 'ui/notify/notifier';
+import uiModules from 'ui/modules';
+// borrowed heavily from https://github.com/styfle/copee
+
+let module = uiModules.get('kibana');
+
+module.service('copee', function () {
+  const copee = this;
+
+  copee.urlToClipboard = (url, name) => {
+    const notify = new Notifier({
+      location: `Share ${name}`,
+    });
```

Comment:
Instead of having a `name` string parameter, how about we have a `notifier` parameter that takes a notifier instance? Then the calling code can be responsible for creating the notifier with the correct name, and can re-use it outside of this service. In particular, this would allow `share.js` to easily do its "URL selected" notification if copying fails which is currently gone in this version of the code.